app/services/external_api/pacman_service.rb

```diff
@@ -0,0 +1,149 @@
+      send_pacman_request(request)
+    end
+
+    # Purpose: Gets distribution from distribution id
+    # POST: /package-manager-service/distribution
+    #
+    # takes in distribution_id(string)
+    #
+    # Response: JSON of distribution from Pacman API
+    # Example response can be seen in lib/fakes/pacman_service.rb under 'fake_distribution_response' method
+    def get_distribution_request(distribution_id)
+      request = {
+        endpoint: GET_DISTRIBUTION_ENDPOINT + distribution_id, method: :get
+      }
+      send_pacman_request(request)
+    end
+
+    private
+
+    # Purpose: Builds package request
+    #
+    # takes in file_number(string), name(string), document_reference(json of strings)
+    #
+    # Response: package request hash
+    def package_request(file_number, name, document_reference)
+      request = {
+        body: {
+          fileNumber: file_number,
+          name: name,
+          documentReferences: {
+            id: document_reference[:id],
+            copies: document_reference[:copies]
+          }
+        },
+        headers: HEADERS,
+        endpoint: SEND_PACKAGE_ENDPOINT, method: :post
+      }
+      request
+    end
+
+    # Purpose: Builds distribution request
+    #
+    # takes in package_id(string), recipient(json of strings), destinations(json of strings)
+    #
+    # Response: Distribution request hash
+    def distribution_request(package_id, recipient, destination)
+      request = {
+        body: {
+          communicationPackageId: package_id,
+          recipient: {
+            type: recipient[:type],
+            name: recipient[:name],
+            firstName: recipient[:first_name],
+            middleName: recipient[:middle_name],
+            lastName: recipient[:last_name],
+            participant_id: recipient[:participant_id],
+            poaCode: recipient[:poa_code],
+            claimantStationOfJurisdiction: recipient[:claimant_station_of_jurisdiction]
+          },
+          destinations: {
+            type: destination[:type],
+            addressLine1: destination[:addressLine1],
+            addressLine2: destination[:addressLine2],
+            addressLine3: destination[:addressLine3],
+            addressLine4: destination[:addressLine4],
+            addressLine5: destination[:addressLine5],
+            addressLine6: destination[:addressLine6],
+            treatLine2AsAddressee: destination[:treatLine2AsAddressee],
+            treatLine3AsAddressee: destination[:treatLine3AsAddressee],
+            city: destination[:city],
+            state: destination[:state],
+            postalCode: destination[:postalCode],
+            countryName: destination[:countryName],
+            emailAddress: destination[:emailAddress],
+            phoneNumber: destination[:phoneNumber]
+          }
+        },
+        headers: HEADERS,
+        endpoint: SEND_PACKAGE_ENDPOINT, method: :post
+      }
+      request
```

Comment:
```suggestion
      }.compact
```
Some of these values will be optional depending on the types of recipients and destinations. I think it'd be a good idea to trim any `nil`s from the body before sending it to Pacman.